schema/plugin.json

```diff
@@ -0,0 +1,14 @@
+{
+  "title": "Table of Contents",
+  "description": "Table of contents settings.",
+  "properties": {
+    "collapsibleNotebooks": {
+      "title": "ToC Configuration",
+      "description": "Specifies whether to allow the ToC extension to collapse notebook cells.",
+      "type": "boolean",
+      "default": true
```

Comment:
As this is causing lots of issues, and there seem to require some additional changes on the lab side, how about we make it disabled (false) by default?